js/web/components/ReusableTable.js

```diff
@@ -1028,6 +1105,7 @@ const SearchResultOverflowContainer = styled(Box)(({ theme }) => ({
   width: '70vw',
   textAlign: 'left',
   textOverflow: 'ellipsis',
+  border: '2px solid red',
```

Comment:
dont want this border

also on line 637 `SearchResultOverflowContainer` is redundantly declared, should be removed